ibis/backends/tests/test_client.py

```diff
@@ -1765,7 +1765,11 @@ def test_cross_database_join(con_create_database, monkeypatch):
     ["impala", "pyspark", "trino"], reason="Default constraints are not supported"
 )
 def test_insert_into_table_missing_columns(con, temp_table):
-    ct_sql = f'CREATE TABLE "{temp_table}" ("a" INT DEFAULT 1, "b" INT)'
+    ident = sg.table(
+        temp_table, db=con.current_database, catalog=con.current_catalog, quoted=True
+    )
+    raw_ident = ident.sql("duckdb")
```

Comment:
This is cool, I was wondering if there was a clean/easy way to do this with SQLGlot! Hopefully this gets the cloud tests all passing now. 

Thanks again for your help with this one!